build-tools.Dockerfile

```diff
@@ -292,7 +292,7 @@ WORKDIR /home/nonroot
 
 # Rust
 # Please keep the version of llvm (installed above) in sync with rust llvm (`rustc --version --verbose | grep LLVM`)
-ENV RUSTC_VERSION=1.86.0
+ENV RUSTC_VERSION=1.87.0
```

Comment:
Do we also want to pin RUSTUP now?

Reply:
i trust the rustup authors have learned that they phase in breaking changes more carefully. And if there is breaking change phase ins, they are more easily seen if we always follow the newest version, which takes extra effort with pinned versions, as it's too easy that you just jump three or four releases to meet immediate breakage.